install/overlays/kubernetes/namespaced/kustomization.yaml

```diff
@@ -14,17 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ---------------------------------------------------------------------------
-
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-#
-# * patch-integration-platform.yaml
-# customizes the integration platform custom resource
-# Edit the patch manually to add required configuration
-#
 resources:
-- ../config/samples/bases/camel_v1_integrationplatform.yaml
+- ../../../base
+- ../../../base/config/rbac/namespaced
+
+namespace: default
```

Comment:
I wonder if it is a good thing to have different default namespaces for the `descoped`  (camel-k) and the `namespaced` (default) overlays.
It might be confusing when we say to create the `camel-k` namespace in the documentation.

Reply:
The goal is to have a clear differentiation between a global operator and a namespaced one. The global will have its own namespace and later, when the user run an Integration it goes by default to the default namespace. In a namespaced installation (which we need to figure out if makes sense to keep supporting) we install in the default namespace by default when the user install in that way.

Reply:
I see the idea, it might be a good idea to add some documentation on this or explain that the commands given in the documentation is oriented for `descoped` installation.

Reply:
Yeah, we need to change entirely the documentation to focus more on the global installation.